MockDuck/Sources/MockBundle.swift

```diff
@@ -172,3 +174,32 @@ final class MockBundle {
         }
     }
 }
+
+private extension URLRequest {
+
+    var bodySteamData: Data? {
```

Comment:
Copied from: 
https://stackoverflow.com/questions/36555018/why-is-the-httpbody-of-a-request-inside-an-nsurlprotocol-subclass-always-nil